larray/session.py

```diff
@@ -92,7 +92,10 @@ def read_arrays(self, keys, *args, **kwargs):
         for key in keys:
             if display:
                 print("loading", key, "...", end=' ')
-            dest_key = key.strip('/')
+            if os.path.isfile(key):
+                dest_key = os.path.splitext(os.path.basename(key))[0]
```

Comment:
I don't really like this particular change because it is too specific. It breaks the generic contract that keys are the actual destination (session) keys. I will fix this myself though.